zone/bot.h

```diff
@@ -608,6 +611,7 @@ class Bot : public NPC {
 	void ListBotSpellSettings();
 	void LoadBotSpellSettings();
 	bool UpdateBotSpellSetting(uint16 spell_id, BotSpellSetting* bs);
+	void SetBotEnforceSpellSetting(bool enforcespellsettings, bool save);
```

Comment:
`bool save = false` so it's default and you don't have to send `false` in `SetBotEnforceSpellSetting`.